src/help.js

```diff
@@ -120,17 +121,17 @@ var stringFormatting = function stringFormatting (str) {
  * @returns {object}
  */
 var groupByMarkerGroupName = function groupByMarkerGroupName (robot, commands, markerName) {
```

Comment:
I guess this alphabetic order should appear in the same pull request as function itself.